src/commands/generateApiSpecFromCodeProject.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import generateApiSpecFromCodeProjectPrompt from "../prompts/generateApiSpecFromCodeProject";
+
+export async function generateApiSpecFromCodeProject(context: IActionContext) {
+    const query = generateApiSpecFromCodeProjectPrompt();
```

Comment:
do we need to check that there is already a project opened in vscode?

Reply:
Updated package.json to only show the command when there is folder opened